src/mip/bound_strengthening.jl

```diff
@@ -0,0 +1,91 @@
+@doc raw"""
```

Comment:
Start the docstring with the function signature, and make sure to match the argument names with what is used in the description below.